Anlab.Core/Extensions/StringExtensions.cs

```diff
@@ -13,5 +14,29 @@ public static string SafeToUpper(this string value)
 
             return value.ToUpper(CultureInfo.InvariantCulture);
         }
+
+        public static bool ClearOutSetupPrice(this string value)
+        {
+            if (value == null)
+            {
+                return false;
+            }
+            if (value.Equals("GRNDONLY", StringComparison.OrdinalIgnoreCase))
+            {
+                return true;
+            }
+
+            return false;
+        }
+
+        public static bool IsGroupTest(this string value)
+        {
+            if (value.StartsWith("G-", StringComparison.OrdinalIgnoreCase))

```

Comment:
@jSylvestre I'd use string.StartsWith() in order to avoid dealing w/ nulls

Reply:
@srkirkland What do you mean? Just remove the if?